src/renderer/component/app/view.jsx

```diff
@@ -6,61 +7,83 @@ import ModalRouter from 'modal/modalRouter';
 import ReactModal from 'react-modal';
 import throttle from 'util/throttle';
 
-class App extends React.PureComponent {
+type Props = {
+  alertError: (string | {}) => void,
+  recordScroll: number => void,
+  currentStackIndex: number,
+  currentPageAttributes: { path: string, scrollY: number },
+  pageTitle: ?string,
+};
+
+class App extends React.PureComponent<Props> {
   constructor() {
     super();
     this.mainContent = undefined;
+    (this: any).scrollListener = this.scrollListener.bind(this);
```

Comment:
What is `(this: any)` doing here? I noticed this in several places.

Reply:
Without it there are flow issues. Got the fix from here https://github.com/facebook/flow/issues/1517